app/assets/stylesheets/common/base/code_highlighting.scss

```diff
@@ -1,24 +1,25 @@
-/* originally based on github.com style (c) Vasily Polovnyov <vast@whiteants.net> */
-
 p > code,
 li > code,
-pre > code,
 strong > code,
 em > code {
-  font-family: ui-monospace, "Cascadia Mono", "Segoe UI Mono", "Liberation Mono",
-    Menlo, Monaco, Consolas, monospace;
+  padding: 2px 4px;
+  background: var(--inline-code-bg);
+  white-space: pre-wrap;
+  color: var(--primary);
+}
+
+code {
   color: var(--primary-very-high);
   background: var(--hljs-bg);
   border-radius: var(--d-button-border-radius);
   font-size: 14px;
   line-height: calc((13 + 4) / 13);
 }
 
-.hljs {
+pre > code {
```

Comment:
If I read this correctly, `pre > code` now has different padding to `code` under a non-pre parent (2px 4px padding above, 12px here for `pre > code`.

Reply:
I have specified the padding to be different for inline versus "block" code.